static/riot_config.json

```diff
@@ -0,0 +1,38 @@
+{
+    "default_server_config": {
+        "m.homeserver": {
+            "base_url": "https://matrix.messenger.schule",
```

Comment:
Where do we overwrite this per Instance?

Reply:
The homeserver value is overwritten when passing the user session from the backend. The riot_config.json file still has to contain a value to avoid errors in the chat initialization.